decidim-proposals/spec/commands/decidim/comments/create_comment_with_proposal_links_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Comments
+    describe CreateComment do
+      include_context "when creating a comment"
+      it_behaves_like "create comment"
+
+      context "and comment contains a link to a proposal" do
+        let(:prop_feature) { create(:proposal_feature, organization: organization) }
+        let(:source_proposal) { create(:proposal, feature: prop_feature) }
+        let(:linked_proposal) { create(:proposal, feature: prop_feature) }
+        let(:parser_context) { { current_organization: organization } }
+        let(:body) { ::Faker::Lorem.paragraph + " ~#{linked_proposal.id}" }
```

Comment:
Is the `~` character needed here?

Reply:
Yes, indeed. It is the symbol that marks the start of a proposal id.